blocks/is-false/index.php

```diff
@@ -27,36 +32,10 @@ function wp_conditional_blocks_is_false_block_init(): void {
  * Short-circuit the display of blocks inside if the outer condition isn't false.
  *
  * @param string|null   $pre_render   The pre-rendered content. Default null.
- * @param WP_Block      $parsed_block The block being rendered.
+ * @param mixed[]       $parsed_block The block being rendered.
```

Comment:
Can `$parsed_block` contain other items in the array besides `WP_Block` objects? What are they?

Reply:
Parsed block is an array containing (at times) the `blockName`, `attrs`, `innerBlocks`, `innerHTML`, and `innerContent` of the block we are rendering. Its never a `WP_Block` object, as far as I can tell.